src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -289,7 +289,7 @@
     long-summary: This address must not be in any Subnet IP ranges, or the Kubernetes service address range. For example, 172.17.0.1/16.
   - name: --load-balancer-sku
     type: string
-    short-summary: Azure Load Balancer SKU selection for your cluster. basic or standard.
+    short-summary: Azure Load Balancer SKU selection for your cluster. basic or standard. Defaults to 'Standard'
```

Comment:
The actual value sent to sever is `standard` (case sensitive). Although the cli will take care of converting the case, it is still recommended that the user enter lowercase.